globals.go

```diff
@@ -666,6 +667,18 @@ var (
 		Description: "No project settings permissions",
 		Value:       0,
 	}
+	AnnotationsModify = PermissionLevel{
+		Description: "Modify annotations",
+		Value:       20,
+	}
+	AnnotationsView = PermissionLevel{
+		Description: "View annotations",
+		Value:       10,
+	}
+	AnnotationsNone = PermissionLevel{
```

Comment:
is there a scenario where someone shouldn't even be able to view annotations?

Reply:
Not currently. But we do have this same option for logs, patches, distros, hosts, tasks, and project settings (i.e. I think every other permission) so it seemed consistent to define it.

Reply:
Non users obviously shouldn't be able to view, so even if we didn't explicitly define this permission it is implicit.

Reply:
oh good point, not logged in users can see the task page but probably shouldn't be able to see annotations